src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -319,6 +319,42 @@ valid_file_mem_backends = @DEFVALIDFILEMEMBACKENDS@
 # Default 0-sized value means unlimited rate.
 #tx_rate_limiter_max_rate = 0
 
+# pvpanic device is a simulated ISA device, through which a guest panic
+# event is sent to qemu, and a QMP GUEST_PANICKED event is generated.
+# This allows management apps to be notified and respond to the event.
+#
+# Note:
+#   To enable pvpanic device, you must set
+#   CONFIG_PVPANIC=y for your kernel.
+#pvpanic_enabled=false
```

Comment:
This PR adds a lot of config options. I think it would make sense to create a table in a markdown document (create a new one if necessary) that summarises the "valid combinations" of these options.